packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -615,6 +625,10 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
     final _ReorderableItemState item = _items[_dragIndex!]!;
     item.dragging = true;
     item.rebuild();
+    _dragStartTransitionComplete = false;
+    SchedulerBinding.instance!.addPostFrameCallback((Duration duration) {
```

Comment:
Wouldn't it be enough to unconditionally set _dartStartTransitionComplete to true after the first time we test it?